package/yast2-theme.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Jun  4 14:35:42 UTC 2020 - Michael Vetter <mvetter@suse.com>
+
+- Add icon for sway pattern
+  See https://github.com/denisok/openSUSEway/issues/1
```

Comment:
I'm not sure if the OBS bots that check for references for bugzilla, Jira, Github, etc. will be satisfied with this format. I'm not sure, but I guess the entry should contain something like `gh#denisok/openSUSEway#1`

Reply:
Personally I always use the full URL in the changes file since probably users read it and can easily click on it. I'm not sure whether in this case it actually mattes to use whether a bot can recognize this URL. But I can change it. Would be okay with having both styles or would you prefer just the short notation?

Reply:
Both is fine

Reply:
Updated!

Reply:
Unfortunately, something was wrong [here](https://ci.opensuse.org/job/yast-yast-theme-master/32/console)

> Stopping, missing new bugzilla or fate entry in the *.changes file.

because of our [Checking for new IDs](https://github.com/openSUSE/packaging_rake_tasks/blob/e6c388b0449a4f0b0fa25602b12247131db2163c/lib/tasks/check_changelog.rake#L52) rake task.

Seem that although @ancorgs was right with the [abbreviation](https://en.opensuse.org/openSUSE:Packaging_Patches_guidelines#Current_set_of_abbreviations) for a Github issue, the task [ID_MATCHERS](https://github.com/openSUSE/packaging_rake_tasks/blob/e6c388b0449a4f0b0fa25602b12247131db2163c/lib/tasks/check_changelog.rake#L21-L49) are **not** in sync with [OBS Issue Trackers](https://github.com/openSUSE/open-build-service/blob/4dcfc836facce35e12d9e58c685e3f416654ac64/src/api/db/seeds.rb#L164).

I'm not sure, but I believe we (the YaST team) were discussing about it back in the past. @mvidner , @jreidinger,  could you help here?

Reply:
I don't see any reason to not add github to our ID_MATCHERS and then retry. I don't remember the details of any conversation about that.

Reply:
Don't me wrong, I'm not saying there is a reason to not having the github tag in our matchers. In my opinion, we must add it. 

About the details, I don't remember them either. But I believe we talked about such sync at the time of adding the tag for Jira. Anyway, it was just a comment just in in case it could bell any ring about the reasons to have or not those matchers synced.

Reply:
Any news here? Would be awesome to have the icon for the openSUSEway pattern :)

Reply:
Today we fixed our tooling and now [the request was submitted successfully](#issuecomment-648223240).

Thanks!